vocab/vocab.csv

```diff
@@ -1166,14 +1166,43 @@ CP14SEPM-SBI01,Coastal Pioneer MAB,Southeastern Profiler Mooring,Surface Buoy,,,
+CP15MOAS-GL379,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,,,,0,1000
+CP15MOAS-GL379-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-GL379-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP15MOAS-GL379-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-GL379-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-GL379-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-GL379-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 379,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-GL387,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,,,,0,1000
+CP15MOAS-GL387-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP15MOAS-GL387-01-ADCPAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP15MOAS-GL387-02-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP15MOAS-GL387-03-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP15MOAS-GL387-04-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP15MOAS-GL387-05-PARADM000,Coastal Pioneer MAB,Mobile Assets,Coastal Glider 387,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP15MOAS-PG583,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 583,,,,0,200
+CP15MOAS-PG583-00-ENG000000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 583,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,200
+CP15MOAS-PG583-01-CTDGVM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 583,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
+CP15MOAS-PG583-02-DOSTAM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 583,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
+CP15MOAS-PG583-03-FLORTM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 583,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,200
+CP15MOAS-PG583-04-FLORTO000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 583,3-Wavelength Fluorometer,WET Labs,ECO BB3,0,200
+CP15MOAS-PG583-05-NUTNRM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 583,Nitrate,Satlantic,SUNA V2,0,200
+CP15MOAS-PG583-06-PARADM000,Coastal Pioneer MAB,Mobile Assets,Profiling Glider 583,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
```

Comment:
yeah - that's why I went with what is in the marine infrastructure Ss - figured that the depths could vary by deployment...and in that case the correct depths for a particular deployment would be captured in the deploy csvs.

Reply:
True, 583's depth won't vary by deployment though. That max depth will be 1000m.

Reply:
And since the others could be equipped to go to 1k as well, maybe we put all their max depths to 1k and then the deployment depth will be captured in the deploy csvs like you said

Reply:
I'll update 583 to 1000 m...
not sure about the other two...Could they potentially have 1000 m max depth with a different pump?  I feel like that is 'deployment-specific' and best captured on a case-by-case basis in the deploy csvs.